docs/README.md

```diff
@@ -29,7 +29,7 @@
 
 **Ƭ BufferLike**: _`Buffer` \| [TransformableToBuffer](interfaces/transformabletobuffer.md) \| [PrefixedHexString](#prefixedhexstring) \| `number`_
 
-_Defined in [types.ts:19](https://github.com/alcuadrado/ethereumjs-tx/blob/84f5b82/src/types.ts#L19)_
```

Comment:
I haven't noticed this 😓

Reply:
Oh, to admit, I haven't noticed the personal repo reference either, wondered on first reading what you refer to. I just re-created the docs for a final run (has turned out to be generally advisable before a release, sometimes it's accurate, sometimes it's not) and didn't put much further thought into it or had some final look. 😄

Reply:
Maybe we should only re-create them only before publishing new releases?

Reply:
Yes, in most cases that makes sense and is normally also the current (relatively 😛) established practice.